src/files/rsql-filter-expression.ts

```diff
@@ -209,3 +241,7 @@ export class RSQLFilterExpression {
 export function quote(value: string | boolean): string {
   return `"${value}"`;
 }
+
+export function encodedQuote(value: string | boolean): string {
```

Comment:
Maybe it would be better to move it to common utils file, because I'm affraid that this method in long term can be used everywhere not only for filters.